x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/components/agent_upgrade_modal/index.tsx

```diff
@@ -106,7 +132,7 @@ export const AgentUpgradeAgentModal: React.FunctionComponent<Props> = ({
           {isSingleAgent ? (
             <FormattedMessage
               id="xpack.ingestManager.upgradeAgents.upgradeSingleDescription"
-              defaultMessage='This action will upgrade the selected agent running on "{hostName}" to version {version}. This action cannot be undone. Are you sure you wish to continue?'
+              defaultMessage='This action upgrades the agent running on "{hostName}" to version {version}. You can&#39;t undo this upgrade.'
```

Comment:
this isn't ideal but using a template literal throws an error during i18n check (#81862) and escaping the single quote doesn't work either